### ISSUE TITLE
Loss visualizations

### ISSUE THREAD

**Issue by celoyd** [celoyd/potato#19]
As a sibling ticket to #14:

* Break loss functions out of the training script.
* Make it easy to heatmap where loss is high on output (or arbitrary pairs of images).

Motivation: I’ve seen the ΔEOK loss (#13) behave so weirdly that I suspect there’s either an implementation bug or something about it that I’m misunderstanding, and either way it would be nice to have a way to visualize it.
